src/style.css

```diff
@@ -0,0 +1,35 @@
+html {
+  box-sizing: border-box;
+}
```

Comment:
Set`box-sizing:` in the `*` selector instead

Reply:
If you set box-sizing: border-box in the `*` selector, all elements will inherit this propery.